front-end/src/pages/HomePage.tsx

```diff
@@ -19,6 +32,19 @@ export default function HomePage(props:RouteComponentProps) {
     props.history.push('/authors/'+userSearch);
   }
 
+
+  let postListElToDisplay;
+  if (postEntries === undefined){
+    postListElToDisplay = <p>Loading...</p>;
+  } else if (postEntries.length === 0) {
+    postListElToDisplay = <p>No Entries Found</p>;
+  } else {
+    postListElToDisplay = postEntries.map((post:Post)=>
+      <PostListItem post={post}/>
+    );
+  }
+  console.log(postListElToDisplay)
```

Comment:
remove console.log

Reply:
Let's actually leave these here for now, I though the same thing at first but IMO they should be really useful for debugging

We can ask the TA if they care about debugging statements left in the code this week and remove them before the deliverables if they do care.

Reply:
Sounds good

Reply:
Yeah my bad, bad habit of mine 😅. I'm fine with either way. It might be more useful to leave them in for people new to the codebase, but I should try to limit the amount of white noise going to the logs going forward